ecaudit/src/main/java/com/ericsson/bss/cassandra/ecaudit/entry/AuditEntry.java

```diff
@@ -254,7 +254,7 @@ public Builder basedOn(AuditEntry entry)
             this.resource = entry.getResource();
             this.operation = entry.getOperation();
             this.user = entry.getUser();
-            this.batchId = entry.getBatchId().orElse(null);
+            this.batchId = entry.batchId;
```

Comment:
Internally this class holds a LazyUUID, but the interface (AuditRecord) has not been updated (that would requires a lot more changes, that I don't thing is necessary - SimpleAuditRecord also implements this interface for the chronicle case). 
BUT that leads to this "issue"; can't no longer do:
this.batchId = entry.getBatchId()
Since we "delazylize" the UUID in the getBatchId method... 
To use the get-method we would have to create a new LazyUUID, but with the UUID already created... That would be an alternative but I did the simple thing here (but at the same time broke the pattern ;-)